sdk/python/kfp/_client.py

```diff
@@ -352,3 +353,18 @@ def _get_workflow_json(self, run_id):
     workflow = get_run_response.pipeline_runtime.workflow_manifest
     workflow_json = json.loads(workflow)
     return workflow_json
+
+  def upload_pipeline(self, pipeline_package_path):
+    """Uploads the pipeline to the Kubeflow Pipelines cluster.
+    Args:
+      pipeline_package_path: Local path to the pipeline package.
+    Returns:
+      Server response object containing pipleine id and other information.
+    """
+
+    response = self._upload_api.upload_pipeline(pipeline_package_path)
+    if self._is_ipython():
+      import IPython
+      html = 'Pipeline link <a href=/pipeline/pipeline?id=%s>here</a>' % response.id
```

Comment:
Could you use _get_url_prefix to build the UI path?

Reply:
Done.